HISTORY.md

```diff
@@ -1,3 +1,5 @@
+## 2.9.0
```

Comment:
If this is a bug fix should be patch version?

Reply:
@nsaenz7 - This PR is to publish persistor 2.8.0 changes. So I would not mark this as a bug fix.